doc/HOWTO/documenting-macros.md

```diff
@@ -0,0 +1,54 @@
+Documenting Macros
+==================
+
+The `make doc-nits` command (or the `check-docs` action) might tell
+you something like:
```

Comment:
Since it is actually much more common that new public functions are added than public macros, it would be great if you wouldn't restrict yourself to the case of macros only, but explain both cases together in a single HOWTO `documenting-functions-and-macros.md`. The process is essentially the same, with only minor differences.

You could also add a short intro with a rationale, like:


> Documenting public Functions and Macros
> =======================================

> In the last years, the OpenSSL project has strived to improve the quality
> and coverage of the API documentation. A while ago, this goal has been
> turned into an official [documentation-policy]. This policy is actively
> enforced by the `make doc-nits` target resp. `check-docs GitHub action.

[documentation-policy]: https://www.openssl.org/policies/technical/documentation-policy.html

> If you add a new public function or macro to a header file without documenting
> it, it give you an error message like this:


```suggestion
Documenting public Functions and Macros
=======================================

In the last years, the OpenSSL project has strived to improve the quality
and coverage of the API documentation. A while ago, this goal has been
turned into an official [documentation-policy]. This policy is actively
enforced by the `make doc-nits` target resp. `check-docs GitHub action.

[documentation-policy]: https://www.openssl.org/policies/technical/documentation-policy.html

If you add a new public function or macro to a header file without documenting
it, it give you an error message like this:
```